Source/controls/devices/game_controller.h

```diff
@@ -8,6 +8,14 @@
 
 namespace devilution {
 
+enum GamepadLayout {
+	Generic = 0,
+	Xbox,
+	Playstation,
+	Nintendo,
+	Virtual

```

Comment:
Nit: Nether alphabetical, or historic order, feels like some sort of favoritism :D
```suggestion
	Generic = 0,
	Nintendo,
	Playstation,
	Xbox,
	Virtual
```

Reply:
Haha definitely not favoritism, or Nintendo would be first :-D